Orange/data/table.py

```diff
@@ -280,45 +280,77 @@ def from_table(cls, domain, source, row_indices=...):
 
         global _conversion_cache
 
-        def get_columns(row_indices, src_cols, n_rows, dtype=np.float64,
-                        is_sparse=False):
+        def array_transform(x, to_sparse):
```

Comment:
Could these functions be moved someplace else? They seem like general functions that ensure array/column is sparse/dense. Maybe change to a more expressive name.